plugins/accelerated-moe/src/fms_acceleration_moe/framework_plugin_scattermoe.py

```diff
@@ -65,7 +66,7 @@ def augmentation(
         rank, world_size = 0, 1
         if torch.distributed.is_initialized():
             world_size = torch.distributed.get_world_size()
-            rank = torch.distributed.get_rank()
+            rank = int(os.environ["LOCAL_RANK"])
```

Comment:
can we make it consistent and follow the new [style](https://github.com/pytorch/pytorch/pull/126737). 
```suggestion
            # we do not need to use the fallback as this is wrapped in an `is_initialized` block   
            rank = torch.distributed.get_node_local_rank()
```

Reply:
Have included this suggestion thanks.